src/pages/register/data-selects.js

```diff
@@ -0,0 +1,32 @@
+const dataSelects = {
+  day: [],
+  month: [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ],
+  year: [],
+};
+
+function createSelects() {
```

Comment:
Let's make separate functions that we can use inside the date and year picker. The problem here that you pass full array of 1...31 values, which means that user can select for example February 31 date, which is incorrect behavior. Let's make the function which will calculate the days count depending on the month that have been selected, and let's move the years logic to the separate function, which will calculate the years array and return it.